portfolio/src/main/webapp/script.js

```diff
@@ -42,4 +42,18 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
+}
+
+async function loadTasks() {
+    let list = await fetch('/list-tasks').then(response => response.json());
+    for(c in list){ 
+        let reviewsElem = document.getElementById("reviews");
+        let commentElem = document.createElement("DIV");
+        commentElem.appendChild(document.createTextNode(list[c].type+" "+list[c].description));
+        reviewsElem.appendChild(commentElem);
+    }
+}
+async function loginStatus() {
+    let status = await fetch('/status').then(response => response.json());
```

Comment:
Same comment as above :)

Reply:
If I merge commentAuth (which solved this problem) into this branch after my other pull request is closed, would that fix this problem or would there be a conflict in the merge?

Reply:
I think if you addressed this comment in the commentAuth, you can just make a comment here saying "I did this in the other PR", and not have to merge different branches together.

Ideally, the simplest workflow is to have individual branches only merging in with master, and not with each other.

Reply:
In the future though, it's best you address comments in the same PR/branch. Then for your other PR/branch, Jasper or I will make the effort not to make the same comment again